src/pretix/control/templates/pretixcontrol/organizers/customer.html

```diff
@@ -71,6 +71,10 @@ <h3 class="panel-title">
                             <dt>{% trans "Notes" %}</dt>
                             <dd>{{ customer.notes|linebreaks }}</dd>
                         {% endif %}
+                        <dt>{% trans "Lifetime spending" %}</dt>
+                        {% for s in lifetime_spending %}
+                            <dd>{{ s.spending|money:s.currency }}</dd>
+                        {% endfor %}
```

Comment:
Currently this looks a little weird if the customer has not bought anything. I think we should either drop the line entirely in this case, or show a zero (without currency? a little weird,  but how would we determine a currency?)

Reply:
I tried different ways of handling it, but in the end I decided on using a dash, the same as is already in use for "last login" in the same area

Reply:
Ah, just saw this now after I submitted my review. I guess regarding the black dash it is not easy. As I suggested in my review, I like the idea of 0,00 in green `.text-success` without any currency.